src/test/java/org/junitpioneer/jupiter/SystemPropertyExtensionTests.java

```diff
@@ -247,15 +273,15 @@ void setForTestMethod() {
 			@AfterAll
 			@ReadsSystemProperty
 			void resetAfterTestMethodExecution() {
-				assertThat(System.getProperty("A")).isEqualTo("old A");
+				assertThat(System.getProperty("A")).isEqualTo("newer A");
```

Comment:
Isn't this (and line 258) a breaking change? (Same for `EnvironmentVariableExtensionTests.java`.)

Reply:
Probably? 🤷 

I would like to pick this up again, but I completely forgot where we stand.

Reply:
Finally a good topic to discuss during a maintainer meeting, I guess? 😅